src/dataregistry/query.py

```diff
@@ -361,3 +368,35 @@ def gen_filter(self, property_name, bin_op, value):
         """
 
         return Filter(property_name, bin_op, value)
+
+    def get_dataset_absolute_path(self, dataset_id, schema=None):
+        """
+        Return full absolute path of specified dataset
+
+        Parameters
+        ----------
+        dataset_id : int
+            Identifies dataset
+        schema : str
+            Defaults to current schema, but may also be "production"
+
+        Returns
+        -------
+        abs_path : str
+        """
+
+        if not schema:
+            schema = self._schema
+
+        # Fetch absolute path, owner type and owner for the dataset
+        if schema != self._schema:
+            raise DataRegistryNYI("schema != default is not yet supported")
+
+        results = self.find_datasets(property_names=['dataset.owner_type',
+                                                     'dataset.owner',
+                                                     'dataset.relative_path'],
+                                     filters=[('dataset.dataset_id', '==',
+                                               dataset_id)])
+        row = results.fetchone()
```

Comment:
should this have a check to see if the dataset was found, will crash otherwise

Reply:
Yes, thanks for finding this.
Should it raise an exception or just issue a warning? I'll go with the latter for now.

Reply:
I think since it will crash anyway, it might as well be an exception to make it clear the reason for the crash